### ISSUE TITLE
[Bug] 'apple-touch-icon' and 'mask-icon' being flagged as invalid values

### ISSUE THREAD

**Issue by hyoretsu** [yannickcr/eslint-plugin-react#3132]
I'm pretty sure it was added recenty, but this should be a bug right? Even MDN which the [rule's page](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/no-invalid-html-attribute.md) tells us to check mentions how iOS doesn't use `icon`, but instead `apple-touch-icon`. `mask-icon` is also used by them on Safari pinned tabs.

**Comment by ljharb** [comment]
Could you provide the exact code being warned, and the exact warning output on the command line?

**Comment by hyoretsu** [comment]
```html
<link rel="apple-touch-icon" sizes="60x60" href="apple-touch-icon-60x60.png" />
<link rel="apple-touch-icon" sizes="76x76" href="apple-touch-icon-76x76.png" />
<link rel="apple-touch-icon" sizes="120x120" href="apple-touch-icon-120x120.png" />
<link rel="apple-touch-icon" sizes="152x152" href="apple-touch-icon-152x152.png" />
<link rel="apple-touch-icon" sizes="180x180" href="apple-touch-icon-180x180.png" />
<link rel="mask-icon" href="/safari-pinned-tab.svg" color="#fff" />
```
This being inside next/document's `<Html><Head /><Html>`
`“apple-touch-icon” is never a valid “rel” attribute value. eslint(react/no-invalid-html-attribute)`
`“mask-icon” is never a valid “rel” attribute value. eslint(react/no-invalid-html-attribute)`

**Comment by gabor-at-reed** [comment]
I've checked this, and the issue still occurs with [v7.30.1](https://github.com/jsx-eslint/eslint-plugin-react/releases/tag/v7.30.1) for me.

**Comment by ljharb** [comment]
@gabor-at-reed we have passing tests, so if you would file a new issue with a repro repo that’d be great.
